doc/src/cylc-user-guide/cug.tex

```diff
@@ -4379,6 +4379,7 @@ \subsubsection{Parameter Expansion}
     [[parameters]]
         obs = ship, buoy, plane
         run = 1..5  # 1, 2, 3, 4, 5
+        idx = 1..9^2  # 1, 3, 5, 7, 9
```

Comment:
The `..` operator is a well known operator in various languages including Perl and Ruby, but none of them have thought about steps.

I don't like `:` or `;` for the same reason as `^` - both are well known operators that mean something else.